cypress/integration/tests/developer/applicationinventory/assessment/filter.test.ts

```
@@ -152,7 +152,8 @@ describe("Application inventory filter validations", { tags: "@tier2" }, functio
         clickByText(button, clearAllFilters);
     });
 
-    it("Business service filter validations", function () {
+    it.skip("Business service filter validations", function () {
```

let the test fail instead of skipping it as it's a bug.